portfolio1.css

```diff
@@ -0,0 +1,141 @@
+body {
+	background-color: #761E1E;
+	color: white;
```

Comment:
use #ffffff instead of white.
you should be consistent with you color type. if hex use hex everywhere